src/frontends/tensorflow/src/input_model.hpp

```diff
@@ -39,6 +39,7 @@ class InputModel : public ov::frontend::InputModel {
     void set_partial_shape(const ov::frontend::Place::Ptr& place, const ov::PartialShape&) override;
     ov::PartialShape get_partial_shape(const ov::frontend::Place::Ptr& place) const override;
     void set_element_type(const ov::frontend::Place::Ptr& place, const ov::element::Type&) override;
+    ov::element::Type get_element_type(const ov::frontend::Place::Ptr& place) const;
```

Comment:
```suggestion
    ov::element::Type get_element_type(const ov::frontend::Place::Ptr& place) const override;
```

Reply:
it looks like we need to extend the base class with this method

Reply:
@itikhono, the base class is extended with this method